travis.yml

```diff
@@ -6,7 +6,7 @@ git:
   depth: false
 matrix:
   include:
-  - go: 1.13.3
+  - go: 1.15
```

Comment:
We no longer use travis for builds right?

Reply:
Perhaps - I wasn't sure whether it had been turned off everywhere.

As far as csi-release-tools is concerned, there might still be some repo which depends on TravisCI, therefore travis.yml is still present and defines the Go version also for Prow (as before):
https://github.com/kubernetes-csi/csi-release-tools/blob/d8a2530054bac287ba9a290f7add8a3423d56f4c/prow.sh#L96-L107